blocks/form/form.js

```diff
@@ -367,6 +368,7 @@ async function createForm(formURL) {
   });
   groupFieldsByFieldSet(form);
   decorateFile(form);
+  if (config.layout === 'wizard') { decorateWizard(form); }
```

Comment:
Instead of configuration we should be using form block style name
`
Form (wizard)
`